### PR TITLE
74-3 & 4 Review

### PR DESCRIPTION
**Pull request by davidtai**
Pass with notes to add simple AWS deployment instructions because this application does not run outside of AWS.  The OP Dr. Adel is addressing this right now.

### CODE REVIEW

**Review by mrkara — COMMENTED** [review]
The Milestone Details part also seems a bit odd, but I don't have any specific comments on that.

File: Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+Requirement | Finding
+------------ | -------------
+OSS contribution best practices | PASS
+
+# Coding Standards
+
+## General Observations
+
+The project
```
[the remark this answers is not in the document]
Comment (davidtai) [reply Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md]: Fixed missing paragraph.

File: Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+Module not found: Can't resolve './aws-exports' in '/mnt/b/projects/start/src'
+```
+
+While its pretty simple to deploy a node.js AWS service and AWS amplify backend service, the documentation was lacking.  However, the AWS deployment artifacts were unremarkable and followed standard AWS conventions so will pass with notes asking for explicit AWS deployment instructions.
+
+The production deployment had features that worked as expected with the casper signer and verified on chain.
+
+## Overall Impression of usage testing
+
+Usage testing involved running through manual testing scripts.  The reviewer did not find any functional issues with the testing scripts.  The reviewer does note that there are some english text sill visible with the Bengali translation.  The reviewer does not have the expertise to judge whether or not having mixed English and Bengali text is considered sufficient translation in that particular region.
```
[the remark this answers is not in the document]
Comment (davidtai) [reply Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md]: Fixed typo

File: Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+------------ | -------------
+https://github.com/dlndao/start/ | 7700b2e
+
+# Install & Usage Testing Procedure and Findings
+
+The installation process is standard for a React + Typescript + node.js application.  The only remarkable thing about the installation process was that `nvm` was required to get it to work becasue it is specifically targeted at version 14 LTS.
+
+This to be expected as the latest node.js LTS that is supported in the targetted runtime is 14 and more modern runtimes would cause backwards comaptibility issues.
+
+The code was completely documented on a function level though there's no overall technical documentation and the function level documentation is rather spartan and could be expanded upon.
```
[the remark this answers is not in the document]
Comment (davidtai) [reply Reviews/74  - Developing DLN Social Smart Contract and DeFi Treasury Investment Policy/3 & 4 - POC Implementation & Closed Beta/README.md]: Spartan -> sparse which is less idiomatic.

**Review by mrkara — APPROVED** [review]
LGTM. Nice work! Thanks!

**Pull request merged**